src/main/resources/data/carpet/tags/blocks/piston_destroy.json

```diff
@@ -0,0 +1,69 @@
+{
+  "replace": false,
+  "values": [
```

Comment:
Are these necessary? I'd assume without being specified to be movable they would have vanilla behavior.

Reply:
Well I added them so you even get some functionality without the datapack.